### DISCUSSION TITLE
Is there a way to quickly switch between presets?

### DISCUSSION THREAD

**Discussion by geoffreyvanwyk** [starship/starship#3317] [Q&A] [answered]
I want to be able to quickly switch between presets, e.g. between the default and the plain text symbols when a Nerd Font is not available. 

One could make **~/.config/startship.toml** a symbolic link to the actual configuration file and then switch the link back and forth.

**Comment by chipbuster** [comment, marked as answer]
Unfortunately, there isn't a quick way to switch presets with a single file at the moment.

Another technique you could use is to point `STARSHIP_CONFIG` to the config you want to use, and then use aliases to switch between them. Because starship reads the config and envar every time, you wouldn't even need to restart the shell to make this take effect.

The ability to pass presets as flags *is* planned for in the long-term, but it'll likely be a while since I'm prioritizing getting out benchmark/performance regression suite set up first.

**Reply by geoffreyvanwyk** [reply]
Thanks @chipbuster.

I used these aliases:

```
alias starship_preset_default='export STARSHIP_CONFIG="${HOME}/.config/starship_preset_default.toml"'
alias starship_preset_plain_text_symbols='export STARSHIP_CONFIG="${HOME}/.config/starship_preset_plain_text_symbols.toml"'
```
